.github/workflows/rust-clippy.yml

```diff
@@ -44,11 +44,19 @@ jobs:
         run:
           cargo clippy
           --all-features
-          --message-format=json | clippy-sarif | tee rust-clippy-results.sarif | sarif-fmt
+          --fix
+          # --message-format=json | clippy-sarif | tee rust-clippy-results.sarif | sarif-fmt
         continue-on-error: true
 
-      - name: Upload analysis results to GitHub
-        uses: github/codeql-action/upload-sarif@v1
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
+        token: ${{ secrets.GITHUB_TOKEN }}
         with:
-          sarif_file: rust-clippy-results.sarif
-          wait-for-processing: true
+          # The new branch
+          branch: ${{ github.event.pull_request.title }}/clippy-fix
```

Comment:
Instead of using the pull request title, maybe use `clippy-fix/<commit hash>`

Reply:
Good idea.